### ISSUE TITLE
Recovery of wallet leads to a transfer of UTXO's to unknown address.

### ISSUE THREAD

**Issue by jediorder21** [BlueWallet/BlueWallet#5912]
## Are you reporting a bug?

Please provide:

* your phone model and OS version
* BlueWallet app version (settings->about->scroll down)
* self-test passes? Open settings->about->scroll down, tap "Run self-test"
* unique ID for our crash reporting service (settings->about->scroll down, tap "copy")

Hello,

We have a serious issue here. I was restoring a bitcoin segwit wallet to a new phone application. The phone is 4 days old purchased directly from the Apple store. The app also downloaded from the Apple app store. I went to restore the wallet with 12 words and I received a pending notification, so we figured it was just taking a few minutes but 4 hours later it was still pending. This is about 11pm at this point. I decide to wait till morning. As you will see a transaction was made at 1:35 am while we were asleep. There has been some major hack happen here with my wallet and I'm freaking out about it. Below is the transaction ID. Please figure out what happened. We having a discord Mod look at the transaction it was done with lightning and is confusing as to what is actually happening.

chevyluv@protonmail.com 
TXID:7e653ce99fcce5c96bf6bacae8307b931dc1e70e7a3bf192b1cb6ab31d79f42

1. IPhone 256GB blue model # MTM73LL/A iOS version 17.0.3
2. Bluewallet ver 6.4.10 (build 1698244191 master
3. Yes, passed self test
4. userId:631AC057-3932-4B1E-8B0936AFEF3B

**Comment by Overtorment** [comment]
hi, this makes no sense to me. 
first, i cant look up the txid, its not valid: https://mempool.space/tx/7e653ce99fcce5c96bf6bacae8307b931dc1e70e7a3bf192b1cb6ab31d79f42
second, i dont get how lightning is related. our lightning node was decommisioned a long time ago, theres a process to recover your funds if you have some stuck there

**Comment by Overtorment** [comment]
did you receive any "support" while installing the app?

anyway, theres lots of ways to shoot yourself in the foot, like custom keyboards that leak typed text, if you took screenshots of the seed it could get automatically uploaded to cloud etc etc.

at the moment this is an isolated incident, dont see any other theft reports atm

**Comment by jediorder21** [comment]
Hello,

That's strange. I'll send the screenshot of it. When I went to the discord to figure out what to do, i spoke with julieluhringxo, who is a mod. She pulled up the transaction ID and said she saw something that led her to believe it involved lightning. I don't think that was the case now. I had Portlandhodl go over the entire thing with me step by step. He wasn't able to recreate the "pending" state that happened directly after entering the seed. It said ETA 3 hours. I just figured it was reconciling with the wallet software since it was just a phone swap to a brand new phone. Blue was transferred over during migration. I verified it was a valid wallet. It's insane this happened, and we can't figure out how. The seed was only on paper and hidden away, so that's not it. Im freaking out about this. I guess I need to contact support. I thought I was just supposed to go to the guthub. Is there a support email I can send screenshots?
